google/cloud/security/scanner/audit/iap_rules_engine.py

```diff
@@ -270,6 +270,7 @@ def find_violations(self, resource, iap_resource):
                 break
             # pylint: enable=compare-to-zero
 
+        print 'XXX: Returning violations: %r' % violations
```

Comment:
I realize this might be for debugging purposes, but would you like to turn this into a LOGGER.debug at some point?